experimental/GModule/Cohomology.jl

```diff
@@ -749,6 +804,10 @@ function cohomology_group(C::GModule{PermGroup,GrpAbFinGen}, i::Int)
   end
   error("only H^0, H^1 and H^2 are supported")
 end
+#TODO:
+#     versions of fp_group(GrpAbFinGen) and extension
+#     to return PcGroups
+#     (via 1st argument being PcGroup)
```

Comment:
So a way to convert a (finite) GrpAbFinGen into a PcGroup -- we can add that (CC @ThomasBreuer )

Reply:
done